libkbfs/keybase_service_base.go

```diff
@@ -292,7 +292,14 @@ func (k *KeybaseServiceBase) Identify(ctx context.Context, assertion, reason str
 		Reason:        keybase1.IdentifyReason{Reason: reason},
 	}
 	res, err := k.identifyClient.Identify2(ctx, arg)
-	if err != nil {
+	// Identify2 still returns the user-plus-keys data, even if it
```

Comment:
Hmm I guess I'm not understanding something -- the user still has keys we can use, even if he doesn't have a sigchain? Are those keys authenticated some other way?